index.html

```diff
@@ -6,6 +6,60 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
+    <section id="exercise-one">
+        <button id="background-changing-button">Change the color!</button>
+        <input id="color-input" />
+    </section>
+    <br>
```

Comment:
Don't use <br> for margins. Use CSS instead

Reply:
I used <br> just for better view. It wasn't supposed to be any professional code and it was faster to make. Should i change it anyway?

Reply:
Yes. Always aim for professional code.